test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -0,0 +1,128 @@
+package kubernetes
```

Comment:
Sure I see no reason not to eventually have a job that runs those tests for us since it is a validation that the kubernetes release will run on azure. But as it stands I'm not I can think of a scenario where those tests would prove anything that should stop a release of acs-engine (or maybe even the RP). They would, however, stop a release of kubernetes/kubernetes.